### PR TITLE
Wip lesson

### LINKED ISSUE #1: Need to build lesson
see outline in [Readme](https://github.com/learn-co-curriculum/sinatra-activerecord-setup)
due date: 9/15

### PR DESCRIPTION
**Pull request by ipc103**
closes #1 - needs review

Estimated Student's Completion Time: 15 - 25 minutes

Made a separate solution branch - will merge README in once edits are complete

@AnnJohn @SophieDeBenedetto @dfenjves @deniznida

### CODE REVIEW

**Comment by AnnJohn** [comment]
Assigned to @deniznida for review

Deadline- today, 9/14/2015

File: README.md [hunk 1]
```diff
@@ -1,9 +1,159 @@
...
+
+Let's test out our handiwork by creating a `dogs` table with two columns: `name` and `breed`. First, let's create our migration:
+
+```bash
+rake db:create_migration NAME=create_dogs
+```
+You should see the following output:
+
+```bash
+=># db/migrate/20150914201353_create_dogs.rb
```
Comment (deniznida) [inline README.md:112]: should we clarify that the number in front of the migration file will be different than what is shown here?

**Comment by deniznida** [comment]
@ipc103 great job, just added one inline comment and fixed some typos

did NOT merge the typo fixes into solution

**Comment by ipc103** [comment]
thanks @deniznida - added a  line about the timestamp

**Comment by AnnJohn** [comment]
@ipc103  How much time do you think this would take a student to complete?

**Pull request merged**